### PR TITLE
helm: unit-tests affinity nodesel tolerations

### PR DESCRIPTION
**Pull request by VPelmeSHko**
* Add: unittests:
  - nodeSelector
  - affinity
  - tolerations

* Add: initializer:
  - tolerations

* Update:
  - templates/tests/unit-tests.yaml
  - values.yaml

### CODE REVIEW

**Review by alles-klar — APPROVED** [review]

**Comment by alles-klar** [comment]
@VPelmeSHko can you use the dev branch as target branch for your PR please?

**Comment by alles-klar** [comment]
@VPelmeSHko The chart version needs a version bump. The failing unit tests does not seem related to this PR for me.

File: helm/defectdojo/templates/tests/unit-tests.yaml [hunk 1]
```diff
@@ -70,4 +70,16 @@ spec:
               key: DD_CREDENTIAL_AES_256_KEY
       resources:
         {{- toYaml .Values.tests.unitTests.resources | nindent 8 }}
+      {{- with .Values.initializer.nodeSelector }}
```
Comment (cneill) [inline helm/defectdojo/templates/tests/unit-tests.yaml:73]:
```suggestion
      {{- with .Values.tests.unitTests.nodeSelector }}
```

File: helm/defectdojo/templates/tests/unit-tests.yaml [hunk 2]
```diff
@@ -70,4 +70,16 @@ spec:
...
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.initializer.affinity }}
```
Comment (cneill) [inline helm/defectdojo/templates/tests/unit-tests.yaml:77]:
```suggestion
      {{- with .Values.tests.unitTests.affinity }}
```

File: helm/defectdojo/templates/tests/unit-tests.yaml [hunk 3]
```diff
@@ -70,4 +70,16 @@ spec:
...
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.initializer.tolerations }}
```
Comment (cneill) [inline helm/defectdojo/templates/tests/unit-tests.yaml:81]:
```suggestion
      {{- with .Values.tests.unitTests.tolerations }}
```

**Review by cneill — CHANGES_REQUESTED** [review]
Thanks for catching the missing `initializer.tolerations` value. However, the modifications to `unit-tests.yaml` reference the wrong values.

**Comment by Maffooch** [comment]
Closing due to stale

**Pull request closed without merging**